artifacts/operator-artifacts/6-controller.yaml

```diff
@@ -31,6 +31,6 @@ spec:
     spec:
       containers:
         - name: wso2am-controller
-          image: wso2/k8s-wso2am-operator:1.0.0
+          image: rakhitharr/k8s-wso2am-operator:100-betav29
```

Comment:
Use the image as wso2/k8s-wso2am-operator:1.0.1

Reply:
updated the images